main.tex

```diff
@@ -415,9 +421,9 @@ \subsection{Limitations of extortionate behaviour} %TODO check that this is a go
 does exhibit ZDs behaviour and so could be extortionate, however, the best
 response is not uniformly a ZDs. A positive measure of skewness and kurtosis,
 and a mean of 0.34 indicate a heavy tail to the right. Therefore, in several
-cases the strategy is not trying to extort its opponents.
+cases the strategy is not trying to extort its opponents. 
```

Comment:
```suggestion
cases the strategy is not trying to extort its opponents. This highlights the importance of adaptability since the best response strategy against an opponent is rarely (if ever) a unique ZDs.
```